### DISCUSSION TITLE
Where is frame advantage displayed for beta on Ryujinx?

### DISCUSSION THREAD

**Discussion by waytehsu** [jugeeya/UltimateTrainingModpack#496] [Q&A] [answered]
Hi, I have turned on the frame advantage toggle within menu. I do not see the frame advantage displayed anywhere. Is this feature working on beta?

Thanks.

**Comment by jugeeya** [comment, marked as answer]
Set shield toggles to Infinite and hit shield. It only works on shield.

**Reply by waytehsu** [reply]
Thank you! The reason it was not showing for me was because I had mash toggle set to something. It has to be set to None for it to work.
